update.json

```diff
@@ -0,0 +1,20 @@
+{
+	"homepage": "https://minecraft.curseforge.com/projects/suppergerrie2s-drone-mod",
+	"1.12.2": {
+		"1.3.1": "changelog here\nsupports special character like \\n"
```

Comment:
You can find the changelog here:
https://minecraft.curseforge.com/projects/suppergerrie2s-drone-mod/files/2607992

Reply:
Added your changelog.